addon/templates/components/common/bs-modal.hbs

```diff
@@ -1,33 +1,35 @@
-{{#if inDom}}
-  {{#maybe-in-element destinationElement _renderInPlace}}
-    {{#component dialogComponent
-      onClose=(action "close")
-      fade=fade
-      showModal=showModal
-      id=modalId
-      keyboard=keyboard
-      size=size
-      backdropClose=backdropClose
-      class=class
-      inDom=inDom
-      paddingLeft=paddingLeft
-      paddingRight=paddingRight
-      centered=(bs-eq position "center")
-    }}
-      {{yield
-        (hash
-          header=(component headerComponent title=title onClose=(action "close"))
-          body=(component bodyComponent)
-          footer=(component footerComponent onClose=(action "close") onSubmit=(action "submit"))
-          close=(action "close")
-          submit=(action "submit")
-        )
-      }}
-    {{/component}}
-
+{{#if this.inDom}}
+  {{#maybe-in-element this.destinationElement this._renderInPlace}}
+    {{#let (component this.dialogComponent) as |Dialog|}}
+      <Dialog
+        @onClose={{action "close"}}
+        @fade={{this.fade}}
+        @showModal={{this.showModal}}
+        @keyboard={{this.keyboard}}
+        @size={{@size}}
+        @backdropClose={{this.backdropClose}}
+        @inDom={{this.inDom}}
+        @paddingLeft={{this.paddingLeft}}
+        @paddingRight={{this.paddingRight}}
+        @centered={{bs-eq this.position "center"}}
+        ...attributes
```

Comment:
Not sure if you are aware of but using `...attributes` and innerHTML components together results in attributes being applied twice: https://ember-twiddle.com/74a339b46d1a87421927992e5a073d1f?openFiles=templates.application.hbs%2Ctemplates.components.my-component.hbs

Reply:
Hm, yes, good catch! I think we could remove the default wrapper element here actually...